specs/opcode/92TLOAD_93TSTORE.md

```diff
@@ -0,0 +1,67 @@
+# TLOAD & TSTORE opcodes
+
+## Variables definition
+
+| Name | Value |
+| - | - |
+| SLOAD_GAS | 100 |
+
+## Constraints
+
+1. opcodeId checks
+   1. opId === OpcodeId(0x5c) for `TLOAD`
+   2. opId === OpcodeId(0x5d) for `TSTORE`
+2. state transition:
+   - gc
+     - `TLOAD`: +7
+       - 4 call_context read
+       - 2 stack operations
+       - 1 transient storage reads
+     - `TSTORE`: +8
+       - 5 call_context read
+       - 2 stack operations
+       - 1 transient storage reads/writes
+   - stack_pointer
+     - `TLOAD`: remains the same
+     - `TSTORE`: -2
+   - pc + 1
+   - reversible_write_counter
+     - `TLOAD`: +0
+     - `TSTORE`: +1 (for transient storage)
+   - gas:
+     - `TLOAD`:
+       - gas + SLOAD_GAS
+     - `SSTORE`:
+       - gas + SLOAD_GAS
+3. lookups:
+   - `TLOAD`: 8 busmapping lookups
+     - call_context:
+       - `tx_id`: Read the `tx_id` for this tx.
+       - `rw_counter_end_of_reversion`: Read the `rw_counter_end` if this tx get reverted.
+       - `is_persistent`: Read if this tx will be reverted.
+       - `callee_address`: Read the `callee_address` of this call.
+     - stack:
+       - `key` is popped off the top of the stack
+       - `value` is pushed on top of the stack
+     - transient storage: The 32 bytes of `value` are read from storage at `key`
+   - `TSTORE`: 10 busmapping lookups
```

Comment:
8 instead of 10?

Reply:
Yes because of no access list and gas refund.